keps/sig-node/2535-ensure-secret-pulled-images/README.md

```diff
@@ -15,9 +15,6 @@
   - [Notes/Constraints/Caveats (Optional)](#notesconstraintscaveats-optional)
   - [Risks and Mitigations](#risks-and-mitigations)
 - [Design Details](#design-details)
-  - [Test Plan](#test-plan)
-      - [Prerequisite testing updates](#prerequisite-testing-updates)
-      - [Unit tests](#unit-tests)
```

Comment:
Why these 2 sectionas are removed from the TOC? I can still see them below.

Reply:
they were duplicated afaict

Reply:
I don't see it in the current content: https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2535-ensure-secret-pulled-images

Reply:
true a rerun of update-toc fixed it, you're right :)